pkg/stub/status.go

```diff
@@ -64,7 +89,8 @@ func (h *Handler) syncOperatorStatus() {
 		}
 	}
 
-	if clusteroperator.ConditionsEqual(oldConditions, co.Status.Conditions) {
+	if clusteroperator.ConditionsEqual(oldConditions, co.Status.Conditions) &&
+		clusteroperator.VersionsEqual(oldVersions, co.Status.Versions) {
```

Comment:
May be also compare old related objects and new co.Status.RelatedObjects

Reply:
Fixed.  Thanks!